### DISCUSSION TITLE
Where to define .well-known/caldav?

### DISCUSSION THREAD

**Discussion by namtscho** [nextcloud/all-in-one#3310] [Questions] [answered]
Hi,

I use AIO behind a nginxProxy. In the Admin I see the hint:

There are some warnings in your system configuration.

    Your web server is not properly configured to resolve "/.well-known/caldav". You can find more information about this in our documentation ↗.
    Your web server is not properly configured to resolve "/.well-known/carddav". You can find more information about this in our documentation ↗._

Where can I solve this problem?

regards

**Comment by szaimen** [comment]
Hi, usually this is handled by AIO by default. Can you show your NginxProxy config?

**Reply by namtscho** [reply]
![07-09-2023_13-52-28](https://github.com/nextcloud/all-in-one/assets/144260053/742a23be-72a9-438f-8c28-6ac18efde61c)

**Reply by szaimen** [reply, marked as answer]
Thanks! Can you remove all of this again and check if it fixes the problem?
![image](https://github.com/nextcloud/all-in-one/assets/42591237/38f0ecf9-b83e-47e4-b398-027309ad37e2)

**Comment by namtscho** [comment]
Thanks a lot, the problem is solved. :-) Copied this from my parallel running old standalone Nextcloud installation.
